tests/src/drives.spec.js

```diff
@@ -1,91 +1,17 @@
 var m = require('mochainon');
+var Promise = require('bluebird');
```

Comment:
'Promise' is defined in a future version of JavaScript. Use a different variable name to avoid migration issues.